src/App.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from './containers/LandingPage/LandingPage';
+import Home from './containers/Home/Home';
+import Meals from './containers/Meals/Meals';
+import Workouts from './containers/Workouts/Workouts';
+import About from './containers/About/About';
+import Blogs from './containers/Blogs/Blogs';
 function App() {
   return (
-    <div>
-      <h1 className="text-3xl text-purple-500">Train To Maintain</h1>
-    </div>
+    <React.Fragment>
```

Comment:
you can change this to 
```suggestion
    <>
```